spec/models/iri_template/url_config_spec.rb

```diff
@@ -0,0 +1,102 @@
+require 'spec_helper'
+
+RSpec.describe Qa::IriTemplate::UrlConfig do
+  let(:url_template) do
+    {
+      "@context" => "http://www.w3.org/ns/hydra/context.jsonld",
+      "@type" => "IriTemplate",
+      "template" => "http://localhost/test_default/search?subauth={?subauth}&query={?query}&param1={?param1}&param2={?param2}",
+      "variableRepresentation" => "BasicRepresentation",
+      "mapping" => [
+        {
+          "@type" => "IriTemplateMapping",
+          "variable" => "query",
+          "property" => "hydra:freetextQuery",
+          "required" => true
+        },
+        {
+          "@type" => "IriTemplateMapping",
+          "variable" => "subauth",
+          "property" => "hydra:freetextQuery",
+          "required" => false,
+          "default" => "search_sub1_name"
+        },
+        {
+          "@type" => "IriTemplateMapping",
+          "variable" => "param1",
+          "property" => "hydra:freetextQuery",
+          "required" => false,
+          "default" => "delta"
+        },
+        {
+          "@type" => "IriTemplateMapping",
+          "variable" => "param2",
+          "property" => "hydra:freetextQuery",
+          "required" => false,
+          "default" => "echo"
+        }
+      ]
+    }.with_indifferent_access
```

Comment:
Do you need the with_indifferent_access here?

Reply:
Removed uses of .with_indifferent_access